### PR TITLE
fix(permutator): fix an issue where lodash was being referred to and not lodash-es

### LINKED ISSUE #507: Fix the issue where we cannot add children in the form generator
Currently clicking the children in the form generator does not result in the children being added to the component, this should work as intended.

### PR DESCRIPTION
**Pull request by janechu**
Resolves #507

### CODE REVIEW

File: packages/fast-permutator/__tests__/test-utils/index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-const get = require('lodash/get'),
-    permutator = require('../../src/index');
+const get = require('lodash-es').get;
```
Comment (chrisdholt) [inline packages/fast-permutator/__tests__/test-utils/index.js]: @janechu Is there a reason for the syntax change here back to dot notation? This ends up increasing our dependency on lodash-es from 9kb to 32kb.

Comment (chrisdholt) [reply packages/fast-permutator/__tests__/test-utils/index.js]: Yep...this resolves it.

**Comment by chrisdholt** [comment]
@janechu can you make one other change? Can you remove the aliasing in the fast-components-react-msft package that moves lodash to lodash-es. It's not needed.

**Comment by janechu** [comment]
Removed the redundant lodash alias

**Pull request merged**